keentools/preferences/ui.py

```diff
@@ -266,61 +263,61 @@ def execute(self, context):
     def cancel(self, context):
         return
 
-    def invoke(self, context, event):
+    def invoke(self, context, _):
```

Comment:
I would prefer to keep full names for arguments of Blender predefined methods. It will help to improve method if you need to change type of a dialog window by proceeding needed parameters. Unnamed parameter is easier to forget.
The same with execute methods.